### PR TITLE
BUGFIX: BB-2123 Add measure type to bullet chart attribute drilling

### PR DESCRIPTION
**Pull request by qduval**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6415/ : SUCCESS in 10m 54s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2197/ : SUCCESS in 5s

**Comment by qduval** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1425.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2208/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2739/ : SUCCESS in 19m 30s

**Comment by qduval** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3332/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/2214/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/2742/ : SUCCESS in 18m 03s

File: src/components/visualizations/utils/test/drilldownEventing.spec.tsx [hunk 1]
```diff
@@ -339,9 +339,9 @@ describe("Drilldown Eventing", () => {
             jest.runAllTimers();
 
             expect(target.dispatchEvent.mock.calls[0][0].detail.drillContext.points).toEqual([
-                { intersection: [], x: 1, y: 100 },
```
Comment (tomassatinsky) [inline src/components/visualizations/utils/test/drilldownEventing.spec.tsx]: I see two functions changed but just one unit test extended. Is it correct? Aren't we missing some new tests?

File: src/components/visualizations/utils/drilldownEventing.ts [hunk 2]
```diff
@@ -92,16 +92,21 @@ const getElementChartType = (chartType: ChartType, point: IHighchartsPointObject
 };
 
 const getDrillPoint = (chartType: ChartType) => (point: IHighchartsPointObject): IDrillPointExtended => {
-    const customProps: Partial<IDrillPointBase> = isComboChart(chartType)
+    const comboChartProps: Partial<IDrillPointBase> = isComboChart(chartType)
```
Comment (tomassatinsky) [inline src/components/visualizations/utils/drilldownEventing.ts]: minor: I would prefer just a single prop set depending on the chart type

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by tomassatinsky** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3337/ : SUCCESS in 16m 24s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6435/ : SUCCESS in 8m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1545/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2214/ : SUCCESS in 23m 40s

**Pull request merged**